src/testers.jl

```diff
@@ -188,6 +188,20 @@ function _test_inferred(f, args...; kwargs...)
     end
 end
 
+"""
+    _is_typestable(f, args...; kwargs...) -> Bool
+
+Return whether `f(args...; kwargs...)` is type-stable.
+"""
+function _is_typestable(f, args...; kwargs...)
```

Comment:
Technically this is not checking type-stability, it is checking inferability.
Julia doesn't promise that all type-stable things will infer.